test/dummy/app/models/mongoid/all_fields.rb

```diff
@@ -0,0 +1,77 @@
+module AllFields
```

Comment:
coveralls is decreasing because of this file. When I added this file I thought that having a single file loading all module fields could be useful. It can be excluded from coverage till it will be really used. @olbrich what do you think?

Reply:
@scicco if we don't use it, we should probably delete it.